authentication/loginPage/login.css

```diff
@@ -0,0 +1,267 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    background-color: var(--lt-cream);
+    font-family: 'Arima Madurai', cursive;
+
+}
+
+header {
+    z-index: 5;
+    padding: 1rem;
+    top: 0;
+    position: sticky;
+    top: 0;
+    width: 100%;
+    background-color: var(--dk-cream);
+}
+
+.flex-sa {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    align-items: center
+}
+
+.flex-gap {
+    gap: 0.1rem;
+}
+
+.block {
+    display: inline-block;
+}
+
+.vilvah-green {
+    color: var(--md-green);
+}
+
+.arima_madurai {
+    font-family: 'Arima Madurai', cursive;
+}
+
+.arima_madurai_100 {
+    font-family: 'Arima Madurai', cursive;
+    font-weight: 100;
+}
+
+.input-items {
+    padding: 0.6rem;
+    padding-right: 4rem;
```

Comment:
you can use like that (padding: 0.6rem 4rem 0.6rem 0rem)

Reply:
ok, i'll make the changes